validation/custom_constraint.rst

```diff
@@ -66,7 +66,7 @@ The validator class is also simple, and only has one required method ``validate(
         public function validate($value, Constraint $constraint)
         {
             if (!$constraint instanceof ContainsAlphanumeric) {
-               throw new UnexpectedTypeException($constraint, ContainsAlphanumeric::class);
+                throw new UnexpectedTypeException($constraint, ContainsAlphanumeric::class);
```

Comment:
This cs fix fix should be backported in 3.4.

Reply:
done in db318b503272248558a0bf1d1b431e4535784fe7